scripts/postInstall.sh

```diff
@@ -0,0 +1,2 @@
+mkdir node_modules/multisig-wallet-gnosis node_modules/multisig-wallet-gnosis/contracts 
+curl https://raw.githubusercontent.com/gnosis/MultiSigWallet/master/contracts/MultiSigWallet.sol -o ./node_modules/multisig-wallet-gnosis/contracts/MultiSigWallet.sol
```

Comment:
Ivan Rukhavets [4:55 PM]
3. Downloading multisig with curl from github seems like a huge security issue
So what we propose is to use the unchanged Gnosis contract and deploy it separately from other contracts

See this branch https://github.com/ambrosus/ambrosus-node-contracts/tree/multisig_actions

Reply:
MultiSigWallet.sol added separatly.